azure-pipelines.yml

```diff
@@ -42,277 +33,239 @@ resources:
+                $(_InternalBuildArgs)
+              displayName: Windows Build / Publish
+            - task: ComponentGovernanceComponentDetection@0
+              displayName: Component Governance scan
 
-- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-  - stage: Create_BAR_ID_Tag
-    displayName: Create BAR ID Tag
-    condition: succeededOrFailed()
-    jobs:
-    - template: /eng/common/templates/job/job.yml
-      parameters:
-        name: Create_BAR_ID_Tag
-        displayName: Create BAR ID Tag
-        pool: 
-          name: $(PoolProvider)
-          demands: ImageOverride -equals windows.vs2022.amd64
-        variables:
-          - group: Publish-Build-Assets
-        steps:
-          - checkout: self
-            clean: true
-          - powershell: eng/create-baridtag.ps1
-              -barToken $(MaestroAccessToken)
-            displayName: Create BAR ID Tag
-            name: Create_BAR_ID_Tag
-  - template: eng\common\templates\post-build\post-build.yml
-    parameters:
-      publishingInfraVersion: 3
-      # Symbol validation isn't being very reliable lately. This should be enabled back
-      # once this issue is resolved: https://github.com/dotnet/arcade/issues/2871
-      enableSymbolValidation: false
-      enableSourceLinkValidation: true
-      # This is to enable SDL runs part of Post-Build Validation Stage
-      SDLValidationParameters:
-        enable: true
-        params: ' -SourceToolsList @("policheck","credscan")
-        -TsaInstanceURL $(_TsaInstanceURL)
-        -TsaProjectName $(_TsaProjectName)
-        -TsaNotificationEmail $(_TsaNotificationEmail)
-        -TsaCodebaseAdmin $(_TsaCodebaseAdmin)
-        -TsaBugAreaPath $(_TsaBugAreaPath)
-        -TsaIterationPath $(_TsaIterationPath)
-        -TsaRepositoryName "Arcade-Validation"
-        -TsaCodebaseName "Arcade-Validation"
-        -TsaPublish $True'
-  - ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/main') }}:
-    - stage: Validate_Publishing
-      displayName: Validate Publishing
-      jobs:
-      - template: /eng/common/templates/job/job.yml
-        parameters:
-          name: Validate_Publishing
-          displayName: Validate Publishing
-          timeoutInMinutes: 240
-          pool:
-            name: $(PoolProvider)
-            demands: ImageOverride -equals windows.vs2022.amd64
-          variables:
-            - group: Publish-Build-Assets
-            - group: DotNetBot-GitHub
-            - name: BARBuildId
-              value: $[ dependencies.setupMaestroVars.outputs['setReleaseVars.BARBuildId'] ]
-            - name: skipComponentGovernanceDetection
-              value: true
-          steps:
-            - template: /eng/common/templates/post-build/setup-maestro-vars.yml
+          - job: Linux
+            container: LinuxContainer
+            pool:
+              name: $(PoolProvider)
+              image: 1es-ubuntu-2204-pt
```

Comment:
nit
```suggestion
              image: 1es-ubuntu-2204
```

Reply:
for selecting the image I have been sticking to the guidance: https://dnceng.visualstudio.com/internal/_wiki/wikis/DNCEng%20Services%20Wiki/1214/1ES-Pipeline-Template-Migration-FAQ?anchor=i%27m-getting-started-on-migrating-my-pipeline%2C-which-pool-should-i-use%3F

is there a particular reason to use the non -pt one? should we update the guidance in that case?

Reply:
the non`-pt` images are currently identical. however the `-pt` ones are just temporary. we're not in a rush but we _will_ delete them

I'll update the guidance.

Reply:
I added some stuff to the guidance just now. Feel free to reword however you like :)

Reply:
much appreciated. I'm going to reword it to _exclude_ the still-incompatible images because that list is shorter 😄 

/fyi you left out the other `windows.vs*` images. that fix (dotnet/dnceng#2080) also rolled out today

Reply:
Ah for some reason I thought that included all the windows images already. Thanks for correcting